test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -368,6 +366,11 @@ def test_sm_pysdk_2(training):
     :param training: training ECR image URI
     """
 
+    _, image_framework_version = get_framework_and_version_from_tag(training)
+    
+    if Version(image_framework_version) == Version("1.5.0"):
+        pytest.skip("sagemaker version < 2.0 is installled for PT 1.5.0 images")
```

Comment:
nit: This is true for 1.5.1 as well.

Reply:
We install `sagemaker>=2,<3` on PT v1.5.1